docs/runtime/nodejs-apis.md

```diff
@@ -510,7 +510,7 @@ The table below lists all globals implemented by Node.js and Bun's current compa
 
 - {% anchor id="node_process" %} [`process`](https://nodejs.org/api/process.html) {% /anchor %}
 - 🟡
-- Missing `process.allowedNodeEnvironmentFlags` `process.channel()` `process.connected` `process.constrainedMemory()` `process.cpuUsage()` `process.debugPort` `process.disconnect()` `process.{get|set}ActiveResourcesInfo()` `process.{get|set}{uid|gid|egid|euid|groups}()` `process.hasUncaughtExceptionCaptureCallback` `process.initGroups()` `process.kill()` `process.listenerCount` `process.memoryUsage()` `process.report` `process.resourceUsage()` `process.setSourceMapsEnabled()` `process.send()`.
+- Missing `process.allowedNodeEnvironmentFlags` `process.channel()` `process.connected` `process.constrainedMemory()` `process.disconnect()` `process.{get|set}ActiveResourcesInfo()` `process.set{uid|gid|egid|euid|groups}()` `process.hasUncaughtExceptionCaptureCallback` `process.initGroups()` `process.report` `process.resourceUsage()` `process.setSourceMapsEnabled()` `process.send()`.
```

Comment:
`process.allowedNodeEnvironmentFlags` is stubbed as an empty object
`process.setSourceMapsEnabled()` is stubbed
`process.getActiveResourcesInfo()` is stubbed
`process.connected` is only defined in a child_process.fork() and that might also apply to process.disconnect and process.send but unsure
`process.get egid|euid|groups|` are all defined